packages/contract_templates/README.md

```diff
@@ -0,0 +1,15 @@
+These templates are used with [abi-gen](https://github.com/0xProject/0x-monorepo/tree/development/packages/abi-gen).
+
+To successfully compile the generated TypeScript contract wrappers, you must:
+* Install the packages on which the main contract template directly depends: `yarn add @0xproject/base-contract @0xproject/sol-compiler @0xproject/types @0xproject/utils @0xproject/web3-wrapper ethers lodash`
```

Comment:
This needs to change, due to [the change from `@0xproject/types` to `ethereum-types`](https://github.com/0xProject/0x-monorepo/commit/a7fc9caacb7b98c7d79333e60ee9862945a67ecc)